node/network/availability-distribution/src/lib.rs

```diff
@@ -843,6 +843,12 @@ impl AvailabilityDistributionSubsystem {
 						);
 					}
 				}
+				FromOverseer::Communication {
+					msg: AvailabilityDistributionMessage::AvailabilityFetchingRequest(_),
+				} => { 
+					// TODO: Implement issue 2306:
+					panic!("To be implemented, see: https://github.com/paritytech/polkadot/issues/2306!");
```

Comment:
this is fine to leave as TODO, but we should not merge to master something that can bring the node down

Reply:
I am already working on the implementation, but right - I should just not make that a panic. That would just be a better default :-)

Reply:
Fixed - just a warning now.